03_bag.ipynb

```diff
@@ -550,75 +521,95 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Pandas is faster than Pure Python.  In the same way, `dask.dataframe` can be faster than `dask.bag`.  We will work more with dataframes later, but from for the bag point of view, they are frequently the end-point of the \"messy\" part of data ingestion - once the data can be made into a data-frame, then complex split-spply-combine logic will become much more straight-forward and efficient.\n",
+    "For the same reasons that Pandas is often faster than pure Python, `dask.dataframe` can be faster than `dask.bag`.  We will work more with DataFrames later, but from for the bag point of view, they are frequently the end-point of the \"messy\" part of data ingestion—once the data can be made into a data-frame, then complex split-apply-combine logic will become much more straight-forward and efficient.\n",
```

Comment:
That's no hyphen, it's an m-dash with no surrounding spaces. Most USA style guide recommend that. Brits are mixed, but often use space-ndash-space. Hyphens should never occur! :-)